src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessToken/Services/IAccessTokenValidator.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading.Tasks;
+
+namespace Altinn.Common.AccessToken.Services
+{
+    /// <summary>
+    /// Interface to validate access token
```

Comment:
Takes some time getting used to, but I'm trying to make the difference between an interface descitpion and class description more obvious. Here is a suggestion.
```suggestion
    /// Defines the methods required for an implementation of an access token validator.
```